manim/mobject/svg/text_mobject.py

```diff
@@ -822,6 +823,22 @@ def gen_chars(self):
                 submobjects_char_index += 1
         return chars
 
+    def gen_words(self):
+        words = VGroup()
+        word = VGroup()
+        submobjects_char_index = 0
+        for char_index in range(self.text.__len__()):
+            if not self.text[char_index] in (" ", "\t", "\n"):
+                word.add(self.submobjects[submobjects_char_index])
+                submobjects_char_index += 1
+            else:
+                if not word.submobjects.__len__() == 0:
+                    words.add(word)
+                word = VGroup()
+        if not word.submobjects.__len__() == 0:
+            words.add(word)
+        return words
```

Comment:
(continuing an above comment regarding calling this function)
```suggestion
        self.words = words
```

Reply:
I was following the precedent of `gen_chars()`, returning them so `self.words` can be set once in the constructor.